warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__shapes_file_present.sql

```diff
@@ -0,0 +1,32 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__schedule_feed_guideline_index') }}
+),
+
+keyed_parse_outcomes AS (
+    SELECT * FROM {{ ref('int_gtfs_schedule__keyed_parse_outcomes')}}
+),
+
+daily_feed_shapes_present AS (
+    SELECT feed_key
+      FROM keyed_parse_outcomes
+     WHERE parse_success
+       AND filename = 'shapes.txt'
```

Comment:
I think for this, it might be safer to use `gtfs_filename = shapes` (`gtfs_filename` is which GTFS file we have decided the data is. So if there was a feed that had a file called ` shapes.txt` with a space and we decided to include that as valid shapes data, it would have `gtfs_filename` = `shapes` even if `filename` had a different value.)

Reply:
Thank you! I saw there were a few options here and was hoping you would guide me to the right one

Reply:
I mean, I could have also asked...